README.md

```diff
@@ -93,15 +95,88 @@ In order to convert the Rmd files to HTML files for Hugo you also need to:
 A shortcut is available through `npm run start:blog`.
 
-#### Adding a new blog post
+## Adding a new blog post
+
+*Prerequisite*: Installed Blog Editor environment from the previous section
+
+### Create new blog file
+
+Create a new file `content/blog` with a naming convention of `YYYY-MM-DD-short-title.Rmd`. The date is the creation date. Copy the template post or another example to start with. *Note*: Starting the file with an underscore (e.g. `_2021-04-20-jj-vaccine.Rmd`), will prohibit that this file is automatically checked by the Github Action CI. This should only be used in case private data is used and thus the public CI cannot built this post.
 
 In case you use new dependencies don't forget to either edit `environment.yml` or `dependencies.R`.
-A Github action should runs when Rmd files changes so it will verify that the blog post can be built.
+A Github action should run when Rmd files changes so it will verify that the blog post can be built.
 However, the converted HTML file along with all generated images are committed to the repository.
 This simplifies the deployment and ensures that we have a blog post even when the API or data changes.
 
+### Edit Blog Header
+
+The header of a blog files contains numerous attributes to be defined. Including name, publication date (`date`), tags, a short summary (at most 150 characters), and a list of authors. The author list `authors` is a list of people keys, see below for how to add people.
+
+The `heroImage` is an optional hero image banner. The image is optional but its thumbnail version is not. However, the template post has good default value for this one. The hero image should be a JPG file with 1120x440 pixels. The thumbnail hero image should be JPG file with 300x200 pixels. The files should be stored in `/content/blog/images`.
+
+The `related` list is a list of related blog links identified by their file name without the Rmd suffix.
+
+### Linking within the blog post
+
+In order to generate links that are relative to the whole website there are two short codes for R Markdown available.
+
+#### Linking to another page
+
+using the `reflink` shortcode. e.g., `r blogdown::shortcode_html("reflink", "2020-09-18-google-survey", "Google")`, will generate a relative link to the blog post with the filename `2020-09-18-google-survey` and the link has a label of `Google`.
+
+#### Linking to the API docs
+
+using the `apireflink` shortcode. e.g., `r blogdown::shortcode_html("apireflink", "api/covidcast.html", "COVIDcast API")` will generate a link to the API docs and its sub page `api/covidcast.html` and will look like: [`COVIDcast API`](https://cmu-delphi.github.io/delphi-epidata/).
+
+### Creating wide figures
+
+In case of a wide figure, one can breakout the layout by adding the `out.extra = 'class="wide-figure"'` extra argument to the corresponding R chunk.
```

Comment:
?

```suggestion
In case of a wide figure, one can break out of the layout by adding the `out.extra = 'class="wide-figure"'` extra argument to the corresponding R chunk.
```